relationships/synthesis/EBPF-CLIENT-to-EBPF_REDIS.yml

```diff
@@ -0,0 +1,82 @@
+relationships:
+  - name: clientConnectsToRedisServerFromServerSpan
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
+      - attribute: eventType
+        anyOf: [ "Span" ]
+      - attribute: redis.req_cmd
+        present: true
+      - attribute: trace_role
+        value: "server"
+    relationship:
```

Comment:
I understand that the relationship is `EBPF-CLIENT calls EBPF-REDIS_SERVER`.

In that case, the source should be:

- Domain: `EBPF`
- Type: `CLIENT`

And the target:

- Domain: `EBPF`
- Type: `REDIS_SERVER`